packages/terra-form-input/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "terra-form-input",
+  "main": "lib/FormInput.js",
+  "version": "0.0.0",
+  "description": "{insert description}",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-form-input",
+    "FormInput",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "devDependencies": {
+    "terra-props-table": "^1.10.1"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
```

Comment:
Make this react 16.2.0, same with react-dom

Reply:
Done. c4abd525098d61a3b1e9238d8ca6a3382473a87b